tests/Testing/TestingTest.php

```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Illuminate\Tests\Foundation;
+
+use Illuminate\Contracts\Foundation\Application;
+use Illuminate\Testing\Testing;
+use Mockery as m;
+use PHPUnit\Framework\TestCase;
+
+class TestingTest extends TestCase
+{
+    protected function setUp(): void
+    {
+        parent::setUp();
+
+        putenv('LARAVEL_PARALLEL_TESTING=1');
```

Comment:
`getenv` and `putenv` should be avoided in favour of `$_SERVER`.

Reply:
Interesting, how come?

Reply:
`getenv` and `putenv` are not threadsafe and leak variables between threads. This is usually not an issue for CLI apps, but can cause big issues for php-fpm apps. Say if you have a production and staging app on the same server, you don't want that the env variables to leak! Usually, what happens is the env variables get randomly unset, rather than actually leaking into other threads, but this often takes people by surprise.

For this reason, it is good practice to just always use the server superglobal for environment variables, and forget anything else exists.

Reply:
Done @GrahamCampbell, thanks!